src/Sulu/Component/Webspace/Analyzer/RequestAnalyzer.php

```diff
@@ -46,7 +46,7 @@ public function analyze(Request $request)
             return;
         }
 
-        $attributes = new RequestAttributes(['host' => $request->getHost(), 'scheme' => $request->getScheme(), 'requestUri' => $request->getRequestUri()]);
```

Comment:
I think the other variables here should also be moved to the `UrlRequestProcessor`, so that only the `RequestProcessor`s are writing into the `RequestAttributes`. However, I was not completely aware of the implications, so I left it like this.